### ISSUE TITLE
Training Strategy

### ISSUE THREAD

**Issue by lzl2040** [Jianing-Qiu/PARNet#2]
I find different datasets contain different numbers of category, so the dimension of the last output layer is different. So please tell me how to train the model with different datasets?

**Comment by Jianing-Qiu** [comment]
Hi there, it is pretty straightforward. All you need to do is to change the output dimension of the last layer to match the number of classes in that dataset. Say the food 101 dataset has 101 classes, then you just need to change the output dimension of the last layer to 101.

**Comment by Jianing-Qiu** [comment]
Sorry, 101, not 100

**Comment by lzl2040** [comment]
> Hi there, it is pretty straightforward. All you need to do is to change the output dimension of the last layer to match the number of classes in that dataset. Say the food 101 dataset has 101 classes, then you just need to change the output dimension of the last layer to 101.

Thansks. I understand what you mean. So the training strategy is, after we train the model on one dataset, we validate the model on that dataset, not after we train the model on all datasets.

**Comment by Jianing-Qiu** [comment]
yes, or you can merge training sets from all datasets into a big one, and do the same for the test sets. Then you can just train one model to recognise a wider range of food categories.

**Comment by lzl2040** [comment]
Thanks very much. The last question is that Do the current training model not shared the weights from the model training on previous dataset?

**Comment by Jianing-Qiu** [comment]
No, it doesn't share weights across datasets. One model for one dataset is the current setting

**Comment by lzl2040** [comment]
Thanks again!
